includes/Dashboard.php

```diff
@@ -262,6 +262,7 @@ public function get_dashboard_settings() {
 				'api'                => [
 					'stories'   => sprintf( '/web-stories/v1/%s', $rest_base ),
 					'media'     => '/web-stories/v1/media',
+					'me'        => '/wp/v2/users/me',
```

Comment:
```suggestion
					'currentUser'        => '/wp/v2/users/me',
```

I think using `currentUser` everywhere instead of `me` is less confusing.

Reply:
Me is the naming conversion that WordPres uses here.

Reply:
For the API endpoint, yes. But we should not use WP naming convention in our editor.

Reply:
Sounds good. I will change those references in the JS code too.